### PR TITLE
Turn positional args into optional args

### PR DESCRIPTION
**Pull request by joelamouche**

### CODE REVIEW

File: test/sign.spec.ts [hunk 1]
```diff
@@ -23,21 +25,21 @@ export async function testSign(command: string): Promise<`0x${string}`> {
 
 export async function testSignCLIPrivateKey(data: string): Promise<`0x${string}`> {
   return testSign(
-    "npm run cli sign ethereum 0x5fb92d6e98884f76de468fa3f6278f8807c48bebc13595d45af5bdc4da702133 " +
+    "npm run cli sign --type ethereum --privateKey 0x5fb92d6e98884f76de468fa3f6278f8807c48bebc13595d45af5bdc4da702133 " +
```
Comment (joelamouche) [inline test/sign.spec.ts:28]: @crystalin I don't understand why if I append the last argument (message) with `--message`, it says "one argument missing"

Comment (crystalin) [reply test/sign.spec.ts:28]: don't you need `--` to pass the argument to the script (instead of passing them to npm) ?

Comment (joelamouche) [reply test/sign.spec.ts:28]: Apparently not. maybe because Im using yarg's command

Comment (joelamouche) [reply test/sign.spec.ts:28]: Oh okay I have figured it out: `--` is indeed needed but I needed to clear the command/usage first

Comment (joelamouche) [reply test/sign.spec.ts:28]: Oh okay I have figured it out: `--` is indeed needed but I needed to clear the command/usage first

**Comment by joelamouche** [comment]
ready for review @crystalin

**Review by crystalin — APPROVED** [review]

**Pull request merged**